gradle-conjure/src/main/java/com/palantir/gradle/conjure/ExtractConjurePlugin.java

```diff
@@ -65,7 +66,14 @@ static ExtractExecutableTask applyConjureJava(Project project) {
     }
 
     static ExtractExecutableTask applyConjureTypeScript(Project project) {
-        return applyAndGet(project, ExtractConjureTypeScriptPlugin.class, ExtractConjureTypeScriptPlugin.TASK_NAME);
+        ExtractExecutableTask result =
+                applyAndGet(project, ExtractConjureTypeScriptPlugin.class, ExtractConjureTypeScriptPlugin.TASK_NAME);
+        // Preserve the conjureTypescript configuration so publishing works
+        if (!Objects.equals(project, project.getRootProject())) {
```

Comment:
We'll be resolving the dependency twice but that should basically be free, so this is probably fine. Think we probably want to be smarter in internal-publish though